website/content/en/docs/advanced-topics/scorecard/custom-tests.md

```diff
@@ -256,17 +256,19 @@ pods created by scorecard.
 
 Scorecard does not deploy service accounts, RBAC resources, or
 namespaces for your test but instead considers these resources
-to be outside its scope.  You can however implement whatever
-service accounts your tests require and then specify
-that service account from the command line using the service-account flag:
+to be outside its scope. You can however specify whichever service account
+your tests require, like `config/rbac/service_account.yaml` in
+Go operator projects, and then specify that service account
+from the command line:
+
 ```console
-$ operator-sdk scorecard <bundle_dir_or_image> --service-account=mycustomsa
+$ operator-sdk scorecard <bundle_dir_or_image> --service-account=my-project-controller-manager
```

Comment:
@estroz just to share. See that this document is for all types. we changed the behaviour for GO and now it deviates from Ansible/Helm. If we do not apply the same changes for Ansible/Helm this doc is no longer valid for them.

c/c @jmrodri

Reply:
Not really, since the service account name is arbitrary.